client/transactions/list/index.tsx

```diff
@@ -204,16 +204,16 @@ const getColumns = (
 		},
 		{
 			key: 'deposit_currency',
-			label: __( 'Deposit Currency', 'woocommerce-payments' ),
-			screenReaderLabel: __( 'Deposit Currency', 'woocommerce-payments' ),
+			label: __( 'Payout Currency', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Payout Currency', 'woocommerce-payments' ),
 			isSortable: true,
 			visible: false,
 		},
 		{
 			key: 'amount',
 			label: __( 'Amount', 'woocommerce-payments' ),
 			screenReaderLabel: __(
-				'Amount in Deposit Curency',
+				'Amount in Payout Currency',
```

Comment:
The WP admin menu links show title case, but not sure if it applies for inner menus and texts. Would love to hear from Magda though if there are any firm guidelines around this. 

![image](https://github.com/user-attachments/assets/5b06c504-be24-4d8e-9e6f-ef7699095867)

Since these issues can be independently tracked, I am merging this PR for now with the feature branch.